hazelcast/src/main/java/com/hazelcast/map/impl/eviction/EvictableCheckerImpl.java

```diff
@@ -38,29 +38,30 @@
  * according to configured {@link com.hazelcast.config.MaxSizeConfig.MaxSizePolicy}
  * to start eviction process.
  *
- * @see com.hazelcast.map.impl.eviction.EvictionOperator#maxSizeChecker
+ * @see EvictionOperatorImpl#evictableChecker
  */
-public class MaxSizeChecker {
+public class EvictableCheckerImpl implements EvictableChecker {
```

Comment:
How many implementations are there? If there is only one, you might consider removing the interface.